aries_cloudagent/wallet/plugin.py

```diff
@@ -29,6 +29,16 @@ def load_postgres_plugin(raise_exc=False):
                 raise OSError(f"Error unable to load postgres wallet storage: {result}")
             else:
                 raise SystemExit(1)
+        if "wallet_scheme" in storage_config:
+            c_config = c_char_p(storage_config.encode('utf-8'))
+            c_credentials = c_char_p(storage_creds.encode('utf-8'))
+            result = stg_lib.init_storagetype(c_config, c_credentials)
+            if result != 0:
+                LOGGER.error("Error unable to configure postgres stg: %s", result)
+                if raise_exc:
+                    raise OSError(f"Error unable to configure postgres stg: {result}")
+                else:
+                    raise SystemExit(1)
```

Comment:
This code is only run once, the first time a PG wallet is opened. Does it need to run more than once if there's different storage types?

Reply:
There should only be one storage type per postgres instance, so the code should only be run once.